yarn.lock

```diff
@@ -1191,6 +1191,14 @@ cssstyle@^3.0.0:
   dependencies:
     rrweb-cssom "^0.6.0"
 
+cypress-multi-reporters@^1.6.4:
```

Comment:
Getting this when I run `yarn` locally, so I'm assuming that means it should be checked in.